docs-chef-io/content/inspec/resources/aws_ecr_repository.md

```diff
@@ -11,7 +11,7 @@ platform = "aws"
     parent = "inspec/resources/aws"
 +++
 
-[\[edit on GitHub\]](https://github.com/inspec/inspec/blob/master/docs-chef-io/content/aws_ecr_repository.md)
+[\[edit on GitHub\]](https://github.com/inspec/inspec/blob/main/docs-chef-io/content/aws_ecr_repository.md)
```

Comment:
We can delete this entire line. This just duplicates a Hugo function that does the same thing.
```suggestion
```